webpage/app/page.tsx

```
@@ -0,0 +1,123 @@
+        <a href="#" className="text-gray-900 font-medium flex flex-row gap-2 items-center"><Image alt="TUM" src={"tum_logo.svg"} width={60} height={1}></Image>MQT QUBOMaker: Pathfinder</a>
+        <a target="_blank" href="https://www.cda.cit.tum.de/research/quantum/" className="text-gray-900 font-medium hidden md:block"><Image alt="MQT" src={"mqt_dark.png"} width={120} height={1} unoptimized></Image></a>
+        <a href="legal" className="text-gray-900 font-medium hidden md:block">Legal Information</a>
+        <a href="legal" className="text-gray-900 font-medium block md:hidden"><Image alt="i" src={"info.png"} width={20} height={1}></Image></a>
+      </nav>
+      <div className="flex flex-col lg:flex-row lg:h-full w-full items-center p-5 justify-between gap-4 lg:overflow-clip">
+        <div className="flex flex-col gap-4 w-full lg:w-auto">
+          {
+            showInfo ? <InfoScreen></InfoScreen> : <GraphView updateAdjacencyMatrix={updateAdjacencyMatrix} upload={doUpload} initialAdjacencyMatrix={adjacencyMatrix}></GraphView>
+          }
+          <div className="flex flex-row justify-around w-full lg:w-auto">
+            <button onClick={() => setShowInfo(!showInfo)} className="border-2 rounded bg-slate-100 p-2 hover:bg-slate-200 active:bg-slate-300">{showInfo ? "Back" : "Help"}</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)